vtex-apps/product-details
# Improve highlight Ui/Ux

#### What is the purpose of this pull request?
As the title said 

#### What problem is this solving?

Improve the poor ux of the product-details schema

#### How should this be manually tested?
You should access the [storefront](https://highlightsprod--storecomponents.myvtex.com/admin/cms/storefront) and search for the `shirt` product.  Then open the schema for `ProductDetails` and change the content and analyze the behavoir. 
- If you choose 'All Specifications' -> all specifications have to displayed
- If you choose 'Groups Specifications', you should type `product tabs`, `group` or `product tabs , group`
- If you choose `Fields Specifications`, you should type the specification name, like `technical Specs`.

#### Types of changes
